DOCUMENTATION.md

```diff
@@ -42,6 +42,13 @@ Create an application. This function only works if you log in using credentials
 
 #### Raises:
     InvalidDeviceType: if device type is not supported.
+
+#### Examples:
+```python
+>>> resin.models.application.create('Edison','Intel Edison')
+Return: dict contains details of new application in the following format.
+... '{"id":9021,"user":{"__deferred":{"uri":"/ewa/user(5397)"},"__id":5397},"app_name":"Edison","git_repository":"g_trong_nghia_nguyen@git.resin.io:g_trong_nghia_nguyen/edison.git","commit":null,"device_type":"intel-edison","__metadata":{"uri":"/ewa/application(9021)","type":""}}'
```

Comment:
Output in Python's REPL isn't prepended by '...'. Just remove this. You may see how this page renders here: https://bitbucket.org/rulemotion/resin-ui/pull-requests/153/reimplemented-actions-view/diff#Lsrc/scripts/modules/pages/dashboard/templates/application/actions.tpl.htmlT27